internal/host/hostcommands/tang_connectivity_check_cmd.go

```diff
@@ -38,6 +38,15 @@ func (c *tangConnectivityCheckCmd) getTangServersFromHostIgnition(host *models.H
 		if err != nil {
 			return nil, err
 		}
+		if luks == nil {
+			return nil, nil
+		}
+		if luks.Clevis == nil {
+			// TODO: not sure how this could happen or whether its possibly valid, for now pretend encryption is disabled just so we avoid a nil pointer dereference
+			// Investigation of this will be carried out in https://issues.redhat.com/browse/MGMT-16721
+			c.log.Warn("luks clevis configuration is missing or incomplete in the host ignition, disk encryption will be assumed to be disabled.")
```

Comment:
Please add the investigation issue ID inside the actual error message: this will save time for people looking at the logs